src/main/java/frc/robot/commands/VisionAutoAlignCommand.java

```diff
@@ -0,0 +1,75 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants.CommandsConstants.VisionAutoAlignConstants;
+import frc.robot.subsystems.SwerveDriveSystem;
+import frc.robot.subsystems.VisionSystem;
+
+public class VisionAutoAlignCommand extends CommandBase {
+    private SwerveDriveSystem m_swerveDrive;
+    private Timer m_timer;
+    private PIDController m_translationXpid = new PIDController(
+            VisionAutoAlignConstants.translationXPID_P, VisionAutoAlignConstants.translationXPID_I,
+            VisionAutoAlignConstants.translationXPID_D);
+    private PIDController m_translationYpid = new PIDController(
+            VisionAutoAlignConstants.translationYPID_P, VisionAutoAlignConstants.translationYPID_I,
+            VisionAutoAlignConstants.translationYPID_D);
+    private PIDController m_rotationPid = new PIDController(VisionAutoAlignConstants.rotationPID_P,
+            VisionAutoAlignConstants.rotationPID_I, VisionAutoAlignConstants.rotationPID_D);
+
+    private VisionSystem m_visionSystem;
+    private double m_targetDistanceMeters = VisionAutoAlignConstants.targetDistanceMeters;
+
+    public VisionAutoAlignCommand(SwerveDriveSystem swerveDrive, VisionSystem visionSystem) {
+        m_swerveDrive = swerveDrive;
+        m_visionSystem = visionSystem;
+        m_timer = new Timer();
+        addRequirements(m_swerveDrive, m_visionSystem);
+
+        m_translationXpid.setTolerance(VisionAutoAlignConstants.errorMarginDistanceX);
+        m_translationYpid.setTolerance(VisionAutoAlignConstants.errorMarginDistanceY);
+        m_rotationPid.setTolerance(VisionAutoAlignConstants.errorMarginRot);
+    }
+
+    @Override
+    public void initialize() {
+        m_timer.start();
+    }
+
+    @Override
+    public void execute() {
+        double xspeed = m_translationXpid.calculate(m_visionSystem.getDistanceMetersX(), 0);
```

Comment:
What is supposed to happen if there are multiple April tags detected?  How do we handle it?
E.g. does this code lock in on ANY April tag ID, or just a specific ID?